src/main/resources/SQLQuery1.sql

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
please delete this from here, it's not 1:1 (should have the same columns/structure) with the entities and also it's working now.
Also align with Alina about this

Reply:
pai daca vad bine asta e practic ce am scris eu prima data

Reply:
@AlinaSastras da, e primul tau fisier. Il pun pe cel nou?

Reply:
ti-l dau tie direct?
ca inca nu am apucat sa il pun nicaieri

Reply:
Pune in utils pe Git pls

Reply:
guys, limitati-va la taskurile voastre pe feature branches si PRs, taskul tau @radumbaciu nu are treaba cu scriptul asta, PR-ul trebuie sa fie curat

Reply:
@radumbaciu fyi
![image](https://user-images.githubusercontent.com/32415039/174171782-c53d8472-b750-4d8f-8afb-4e4b231b5a88.png)